.gitattributes

```diff
@@ -1 +1,2 @@
-*.ipynb linguist-vendored
\ No newline at end of file
+*.ipynb linguist-vendored
+* text eol=lf
```

Comment:
This change simply prevents a minor annoyance when developing on Windows where EOL characters are automatically changed to CRLF.